src/pages/index.js

```diff
@@ -1,7 +1,23 @@
-export default function HomePage() {
+import transactions from '../data/transactions.json'
+
+function TransactionList() {
+  // Sort transactions by date in descending order
+  const sortedTransactions = transactions.sort((a, b) => new Date(b.date) - new Date(a.date));
+
   return (
     <div>
-      <h1>Hello from Next.js</h1>
+      <h1>Transactions</h1>
+      <ul>
+        {sortedTransactions.map((transaction) => (
+          <li key={transaction.id}>
+            <h3>{transaction.category}</h3>
+            <div>{new Date(transaction.date).toLocaleDateString('de-DE')}</div>
+            <div>{transaction.amount.toFixed(2)} {transaction.currency}</div>
```

Comment:
Old suggestion:

Keep correctly rounded amounts in you dataset instead
 <div>{transaction.amount} {transaction.currency}</div>